packaging/windows/nsis/README.md

```diff
@@ -27,13 +27,19 @@ repository.
 
 ## How to build the installer manually
 1) Install NSIS and make sure `makensis` it is available in the `%PATH%`.
-2) Download and decompress the package assets to your machine.
-3) [Build Khiops in Release mode](https://github.com/KhiopsML/khiops/wiki/Building-Khiops)
+2) Download and decompress the package assets to your machine, on `packaging\windows\nsis\assets`
+3) [Build Khiops in Release mode](https://github.com/KhiopsML/khiops/wiki/Building-Khiops), to build the binaries and the jars
 4) In a console, go to the `packaging/windows/nsis` directory and execute
 ```bat
-%REM We assume the package assets were downoladed to packaging\windows\nsis\assets
+%REM We assume the package assets were downloaded to packaging\windows\nsis\assets
+%REM Before building the installer, update the following script arguments, mainly:
+%REM - KHIOPS_VERSION
+%REM - KHIOPS_REDUCED_VERSION
+%REM - KHIOPS_VIZ_INSTALLER_PATH, with last version of the installer available in the assets
+%REM - KHIOPS_COVIZ_INSTALLER_PATH, with last version of the installer available in the assets
+
 makensis ^
-   /DKHIOPS_VERSION=10.2.0-preview ^
+   /DKHIOPS_VERSION=10.2.0-rc.1 ^
    /DKHIOPS_REDUCED_VERSION=10.2.0 ^
```

Comment:
I'm not sure about how these versions should be updated, as they are moving targets according to the ongoing release. 

Perhaps, as this is a readme, we could follow either one of the two approaches listed below:
- we don't change anything to the version numbers themselves, but add a comment that these should be updated according to the pending release version
- we systematically update this to the version of the last pre-release (e.g. 10.2.3-rc.2 in our case).

Either of the two choices seems acceptable to me, although I would lean towards the first one, as it seems less expensive to me (it requires no further maintenance of the README upon each new release).

Reply:
J'ai juste changé "preview" en "rc.1" pour avoir un exemple réaliste de version officiel (à ma connaissance, "preview" n'est pas un suffixe officiel)
Sinon, j'ai déjà ajouté un commentaire "update the following script arguments" dans le sens de la première approche.

Reply:
Done